docs/Tutorials/apiTutorial.md

```diff
@@ -21,158 +21,134 @@ A minimum network configuration must include:
 
-!!! note "Note - MPS API Authorize Method"
-    See the [Authorize Method in the API Documentation](https://app.swaggerhub.com/apis-docs/rbheopenamt/mps/{{ mpsAPI.version }}#/Auth/post_api_v1_authorize){target=_blank} for the structure and other requirements of the Authorize API call used in the following to generate a JWT.
+1. Open a Terminal or Command Prompt.
+2. Copy and paste the example code below into a text editor.
+3. Update the values of the `[IP-Address or FQDN]`, `[MPS_WEB_ADMIN_USER]`, and `[MPS_WEB_ADMIN_PASSWORD]` fields.
 
-1. Navigate to a file directory of your choice.
-2. Create and open a new JavaScript* file with a name of your choice. In this guide, we will refer to it as *generateJWT.js*.
-3. Copy and paste the example code below.
-4. Update the values of the `username`, `password`, and `hostname` keys.
-
-    !!! example "Example POST - generateJWT.js"
-
-        ```javascript hl_lines="4 5 9"
-        const https = require('https')
-        process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0 //For testing withself-signed certs, remove for production
-        let data = JSON.stringify({
-            'username': 'standalone', //Replace with MPS_WEB_ADMIN_USER from .env file or mpsweb stored secret
-            'password': 'G@ppm0ym' //Replace with MPS_WEB_ADMIN_PASSWORD from .env file or mpsweb stored secret
-        })
-
-        const options = {
-            hostname: 'localhost', //Replace 'localhost' with IP Address or FQDN
-            path: '/mps/login/api/v1/authorize',
-            method: 'POST',
-            headers: {
-                'Content-Type': 'application/json'
-            }
-        }
-
-        const req = https.request(options, (res) => {
-            res.setEncoding('utf8')
-            res.on('data', d => {
-                console.log(d)
-            })
-        })
-
-        req.on('error', (e) => {
-            console.error(e)
-        })
-
-        // Write data to request body
-        req.write(data)
-        req.end()
+    === "Linux"
+        ```bash
+        curl --insecure -X POST https://[IP-Address or FQDN]/mps/login/api/v1/  authorize \
+            -H "Content-Type:application/json" \
+            -d "{\"username\":\"[MPS_WEB_ADMIN_USER]\", \"password\":\" [MPS_WEB_ADMIN_PASSWORD]\"}"
+        ```
+    === "Windows"
+        ```bash
+        curl --insecure -X POST https://[IP-Address or FQDN]/mps/login/api/v1/  authorize ^
+            -H "Content-Type:application/json" ^
+            -d "{\"username\":\"[MPS_WEB_ADMIN_USER]\", \"password\":\" [MPS_WEB_ADMIN_PASSWORD]\"}"
         ```
 
-### Execute the REST API
-
-1. Open a Terminal or Command Prompt to execute the call.
-2. Navigate to the directory you saved the generateJWT.js file.
-3. Run the code snippet using node.
+    !!!info "Info - Using the --insecure Flag"
+        Because we are using self-signed certificates for MPS for development purposes, we must supply this flag to force past certificatverification.
```

Comment:
certificate verification? Or again, like the one about URI vs. URL below, am I responding to something intentional?

Like is there some thing called certificatverification?